packages/lesswrong/lib/routes.ts

```diff
@@ -1328,14 +1328,15 @@ addRoute(...forumSelect<Route[]>({
       name: 'inbox',
       path: '/inbox',
       componentName: 'InboxWrapper',
-      title: "Inbox"
+      title: "Inbox",
+      fullscreen: true,
     },
     {
       name: 'conversation',
       path: '/inbox/:_id',
-      componentName: 'ConversationWrapper',
-      title: "Private Conversation",
-      background: "white",
+      componentName: 'InboxWrapper',
+      title: "Inbox",
+      fullscreen: true,
```

Comment:
This is a weird interaction for LW (this makes it so that `/inbox` is the old UI and `/inbox/<id>` is the new UI). I know they want to switch to the nice new UI anyway, so I would vote to just make it the default for everyone. @Stvad Would you be down to do that in this PR? I'm guessing it would be pretty quick, and maybe there are some UI bits that LW would need to clean up afterwards. @lagerros is the LW person to work with here

Reply:
hmm, this change is to actually make it consistent - current codebase default mixes old specific message view and new general inbox view. This change makes both default to a new message view. 

I'm not sure why things are not _currently_ mixed up on LW side - is there a LW specific override somewhere (don't see it in routes but maybe there is a check elsewhere in codebase somewhere)?

Reply:
Maybe try that without the LW overrides and see if @lagerros is happy to ship it like that to LW?

Reply:
Pushed a commit for that. @lagerros here is what it looks like with LW forum type, lmk what you think: 
<img width="1239" alt="image" src="https://github.com/ForumMagnum/ForumMagnum/assets/81347/ac4be0c3-c373-40fa-9b3b-097769739223">
<img width="1184" alt="image" src="https://github.com/ForumMagnum/ForumMagnum/assets/81347/0ac580d8-66e2-437d-ba6b-384f654c9486">

Reply:
@lagerros checking if you've seen this

Reply:
wasn't able to get a hold of @lagerros . will add LW overrides to move this forward